docker-squash.sh

```diff
@@ -75,6 +81,8 @@ get_healthcheck() { parse_config "$1" '.Healthcheck // empty|(
     (.Test[0]|sub("-SHELL";"")) + " " + .Test[1]
 )'; }
 
+DOCKERFILE_SYNTAX="${DOCKERFILE_SYNTAX:-docker/dockerfile:1}"
```

Comment:
I think this line is unnecessary since the environment variable is only used in one place.